src/Indices/IndexManager.php

```diff
@@ -35,6 +35,13 @@ public function close(string $indexName): self
         return $this;
     }
 
+    public function get(string $indexPattern): array
```

Comment:
This method returns a raw result, which is inconsistent with other functionality (see `getAliases` for example). Maybe it would make more sense to add a method called `client` in both `IndexManager` and `DocumentManager` to give users more flexibility when they need something, which is not covered in managers?

```php
// in this case "client" returns IndicesNamespace
$indices = $indexManager->client()->get('*');
```

Reply:
Ah... I see now, that this is needed for the `elastic:migrate:fresh` command. Do we even need a new method for that purpose? We can call `IndexManager::drop($indexName)` with a wildcard, e.g. `$indexManager->drop('*')`

Reply:
I'd say having the `client()` method in the `IndexManager` and `DocumentManager` would be great for developers convenience and flexibility.

> Do we even need a new method for that purpose? We can call `IndexManager::drop($indexName)` with a wildcard, e.g. `$indexManager->drop('*')`

I guess it depends on what you want the `IndexManager` to be. Should it be a handful of convenience methods along with access to the underlying ES PHP client? Or should it have its own API that greatly reduces the complexity of ES index management?

Reply:
Elastic Adapter is just a library, which is created to support Elastic Scout Driver (Plus) and Elastic Migrations. It was not intended, that someone would use it standalone, that's why API is limited.